events.go

```diff
@@ -98,3 +99,13 @@ information is displayed once every 5 seconds.`,
 		group.Wait()
 	},
 }
+
+func containerStatus(container libcontainer.Container) {
```

Comment:
the name of this function does not tell or hint at what it does.  It maybe better to just move this code into the caller or find a better name that explains its checking if the container is not destroyed, etc....